docs/Reduced_Operator_Kernel_build.md

```diff
@@ -0,0 +1,71 @@
+# ONNX Runtime Reduced Operator Kernel build
+
+In order to reduce the compiled binary size of ONNX Runtime (ORT), the operator kernels included in the build can be reduced to just the kernels required for your scenario. 
+
+The kernels to include must first be identified, and secondly the ORT kernel registration source files must be updated to exclude the unused kernels. 
+
+Finally ORT must be manually built.
+
+When building ORT with a reduced set of kernel registrations, `--skip_tests` *MUST* be specified as the kernel reduction will render many of the unit tests invalid. 
```

Comment:
Can this be merged with the [ONNX_Runtime_Format_Model_Usage.md](ONNX_Runtime_Format_Model_Usage.md) to include the build with the build steps there?
OR
Move the build steps here and leave that doc for using ORT for Mobile platform.

Reply:
They should be separate. A reduced ops build is the 'medium' size ORT and is not dependent on the minimal build (which is the 'small' size ORT), so there's only one step to do that. For the minimal build there are multiple steps. 

I could maybe add an example command line to call the reduction script to the minimal build doc so there's less need to fully read this document when doing a minimal build.

Reply:
Added example to ORT Mobile doco